src/HatsModule.sol

```diff
@@ -38,7 +38,7 @@ contract HatsModule is IHatsModule {
   //////////////////////////////////////////////////////////////*/
 
   /// @inheritdoc IHatsModule
-  function setUp(bytes calldata _initData) public {
+  function setUp(bytes calldata _initData) public initializer {
```

Comment:
It is probably a good idea to add a test for this

Reply:
Done: https://github.com/alexkeating/hats-module/pull/3/commits/2f7a6451ef4c14f8e98ed925668ce098a9a03022